### PR TITLE
chore(documentation): Update release flow in collaborator guide md

### PR DESCRIPTION
**Pull request by dzianis-dashkevich**
Update release flow in collaborator guide md

### CODE REVIEW

File: COLLABORATOR_GUIDE.md [hunk 1]
```diff
@@ -283,13 +282,15 @@ git reset --hard upstream/main
 ## video.js releases
 
 Releasing video.js is partially automated through various scripts.
-To do a release, you need a couple of things: npm access, GitHub personal access token.
```
Comment (gkatsev) [inline COLLABORATOR_GUIDE.md:286]: comment(non-blocking,no-change): technically, there isn't even a need for npm access to do a release anymore. Just write access to the repo!

File: COLLABORATOR_GUIDE.md [hunk 2]
```diff
@@ -283,13 +282,15 @@ git reset --hard upstream/main
...
+To do a release, you need npm access.
 
 Releases in video.js are done on npm and GitHub and eventually posted on the CDN.
 These are the instructions for the npm/GitHub releases.
 
 When we do a release, we release it as a `next` tag on npm first and then at least a week later, we promote this release to `latest` on npm.
 
+you can promote it using `npm dist-tag add video.js@<version> <tag>`
```
Comment (gkatsev) [inline COLLABORATOR_GUIDE.md:292]: nit:
```suggestion
You can promote it using `npm dist-tag add video.js@<version> <tag>`.
```

File: COLLABORATOR_GUIDE.md [hunk 3]
```diff
@@ -321,15 +316,14 @@ Most common releases will be either `patch` or `minor`.
...
 
 ```sh
-# for v6
-git clone git@github.com:videojs/video.js.git videojs-6-release
-# for v5
-git clone git@github.com:videojs/video.js.git videojs-5-release
+# for v8
+git clone git@github.com:videojs/video.js.git videojs-8-release
+# for v7
+git clone git@github.com:videojs/video.js.git videojs-7-release
```
Comment (gkatsev) [inline COLLABORATOR_GUIDE.md:326]: 👍🏻

File: COLLABORATOR_GUIDE.md [hunk 4]
```diff
@@ -362,21 +356,13 @@ relies on the commit being available on GitHub.
...
-```
-
-After it's done, verify that the GitHub release has the correct changelog output.
-This is to make sure that the CHANGELOG didn't get garbled and isn't missing pieces.
+After the tag was pushed, GitHub actions will trigger the `release` workflow, which will do the following:
 
-If the GitHub release did not work correctly, such as if the GitHub token was not provided,
-you can run it manually:
+* Publish to npm with `next` or `next-{n}` depending on your current major version.
+* Create GitHub pre-release with changelog and Netlify preview.
```
Comment (gkatsev) [inline COLLABORATOR_GUIDE.md:362]: It'll also automatically create a discussion linked to the GitHub release.

File: COLLABORATOR_GUIDE.md (hunk 1 above)
Comment (dzianis-dashkevich) [reply COLLABORATOR_GUIDE.md:286]: 👍 updated line

File: COLLABORATOR_GUIDE.md (hunk 2 above)
Comment (dzianis-dashkevich) [reply COLLABORATOR_GUIDE.md]: added

File: COLLABORATOR_GUIDE.md (hunk 4 above)
Comment (dzianis-dashkevich) [reply COLLABORATOR_GUIDE.md:362]: added

File: COLLABORATOR_GUIDE.md [hunk 5]
```diff
@@ -362,21 +356,14 @@ relies on the commit being available on GitHub.
...
+* Create a GitHub `releases` discussion linked to the GitHub release.
+* Copy files to the CDN with the AWS CLI (this step requires approval, make sure to ping collaborators chat!)
 
-```sh
-VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=123 node build/gh-release.js --prelease
-```
+And that's it. Congratulations - you've just released a new version of video.js.
```
Comment (gkatsev) [inline COLLABORATOR_GUIDE.md:366]: nit(non-blocking):
```suggestion
And that's it. Congratulations - you've just released a new version of Video.js.
```

Feel free to update all *text* occurrences to `Video.js`.

**Review by gkatsev — APPROVED** [review]

File: COLLABORATOR_GUIDE.md (hunk 5 above)
Comment (dzianis-dashkevich) [reply COLLABORATOR_GUIDE.md]: updated

**Review by misteroneill — APPROVED** [review]

**Pull request merged**